core/client/client.go

```diff
@@ -284,19 +284,35 @@ func (c *Client) newMutation(ctx context.Context, u *tpb.User) (*entry.Mutation,
 	return mutation, nil
 }
 
-// WaitForUserUpdate waits for the STH to be updated, indicating the next epoch has been created,
+// WaitForUserUpdate repeately waits for the mutation to be applied.
```

Comment:
typo. Could remove repeatedly and add "or the context to timeout or be cancelled."

Reply:
done